lib/takeaway.rb

```diff
@@ -0,0 +1,95 @@
+require 'dotenv'
+require './lib/menu'
+require './lib/messaging'
+
+class Takeaway 
+
+attr_reader :order, :bill
+
+	def initialize (menu = Menu.new, message = Message.new )
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)